### DISCUSSION TITLE
Enhancement Idea: Display node IP

### DISCUSSION THREAD

**Discussion by minix1234** [hassio-addons/app-zerotier#48] [Ideas]
First of thank you for this addon, it is well put together (as usual).

# Enhancement Idea

Would it be possible to somehow log or display the zerotier interfaces IP address? I can see it in the zerotier dashboard online, but having it entered into the log messages would be great to help quickly troubleshoot any issues.

One method might be to grab the data from the zerotier command line tool.  `zerotier-cli listnetworks`

![image](https://user-images.githubusercontent.com/48591487/110538525-210eef80-80f2-11eb-804d-c65d6b63f0d9.png)

**Comment by frenck** [comment]
I do like the idea in general, however, I don't know when that information would be complete, available or changed..
